index.html

```diff
@@ -199,7 +199,21 @@ <h2 class="subt">Skills</h2>
           </div>
         </div>
       </section>
-    </main>
-    <footer></footer>
+      <footer class="contact-form">
+        <section class="form-section">
+          <div class="contact-me">
+            <h2 >Contact me</h2>
+            <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+          </div>
+          <form action="send" class="form">
+            <input type="text" class="text" value="Yeremias">
+            <input type="email" class="text" value="natajayanj14@gmail.com">
+            <textarea class="textarea">Write your message here</textarea>
+            <input type="button" class="button" value="Get It Touch">
```

Comment:
Your form is not submitting the data because your `input` type is button instead of submit. Kindly make the necessary changes so that your form sends the data successfully to Formspree 😎

    <input type="submit">

Reply:
Changes done!